exoctk/contam_visibility/visibilityPA.py

```diff
@@ -332,7 +331,6 @@ def using_gtvt(ra, dec, instrument, ephFileName=None, output='bokeh'):
     # Plot formatting
     fig.xaxis.axis_label = 'Date'
     fig.yaxis.axis_label = 'Position Angle (degrees)'
-    fig.y_range = ranges.Range1d(0, 360)

```

Comment:
@jaymedina Again -- this was a change we made purposefully -- please bring it back.